nextflow_schema.json

```diff
@@ -10,24 +10,30 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": [
-                "input"
-            ],
+            "required": ["input", "databases", "outdir"],
             "properties": {
                 "input": {
                     "type": "string",
                     "format": "file-path",
                     "mimetype": "text/csv",
-                    "pattern": "^\\S+\\.csv$",
+                    "pattern": "^\\S+\\.(csv|yaml|yml)$",
```

Comment:
😱 this also nice. I will check how you parse this. Do you use a library that can handle all three or are you implementing this manually?

Reply:
Ah that is left over from when we had PEP support, but this functioanlity has unfortunately had to be removed :\.

I'll move this back to just `csv` again.